workspaces/config/lib/type-defs.js

```diff
@@ -1,10 +1,12 @@
 const nopt = require('nopt')
 
-const { Umask, validate: validateUmask } = require('./umask.js')
+const { validate: validateUmask } = require('./umask.js')
 
-const semver = require('semver')
+class Umask {}
+class Semver {}
```

Comment:
@H4ad the class is meaningless it's just a unique "object" to hang nopt definitions off of.  I opted for the same as `Umask` here instead. Thoughts?